modules/EvseManager/EvseManager.cpp

```diff
@@ -1177,7 +1177,7 @@ void EvseManager::setup_v2h_mode() {
     types::energy::ExternalLimits external_limits;
     types::energy::ScheduleReqEntry target_entry;
     target_entry.timestamp = timestamp;
-    target_entry.limits_to_leaves.total_power_W = powersupply_capabilities.max_import_power_W.value();
+    target_entry.limits_to_leaves.total_power_W = powersupply_capabilities.max_import_power_W.value_or(0);
```

Comment:
Not necessary, as it is checked beforehand whether `max_import_power_W` has a value (line 1152). If this is the case, the function is returned.